wsuwp-embeds.php

```diff
@@ -8,6 +8,30 @@
 Author URI: https://web.wsu.edu/
 */
 
+/**
```

Comment:
Let's put this in the `includes/wsu-embed-youtube.php` file with a note that it was forked from https://github.com/Automattic/jetpack/blob/master/modules/shortcodes.php